bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogManagerForSingleEntryLog.java

```diff
@@ -211,9 +211,7 @@ void flushRotatedLogs() throws IOException {
 
     @Override
     public void close() throws IOException {
-        if (activeLogChannel != null) {
-            activeLogChannel.close();
-        }
+        IOUtils.close(log, activeLogChannel);
```

Comment:
unrelated note.
With current method definition
```java
    public static void close(Logger log, java.io.Closeable... closeables) {
```

java will create new Object[] for single closeable that is passed to the method.
We'll benefit from a variant that takes only one parameter. Do we ever use this method with more than one closeable?

Reply:
> java will create new Object[] for single closeable that is passed to the method.

yes, but it is probably okay since we use this method at shutdown phase.

> We'll benefit from a variant that takes only one parameter.

definitely we will benefit from it. +1